docker/docker-compose-build.yml

```diff
@@ -2,16 +2,15 @@ version: "3.0"
 services:
   mysql:
     image: "mysql:5.7"
-    env_file:
-      .env
+    environment:
```

Comment:
Note: there was no `.env` file, so alas, the docker stuff wasn't working anyway.

I axed the env stuff to avoid some pathing issues in tox